pyproject.toml

```diff
@@ -40,6 +43,18 @@ docs = [
 testing = [
     "ops-scenario>=7.0.5,<8",
 ]
+tracing = [
+    "otlp-json>=0.9.2",
+    # FIXME: decision needed
+    #
+    # On one hand, this instrumenttion provides additional info about pebble
+    # (request URL, status code, post and response data sizes)
```

Comment:
Can't we drop this and just do this ourselves manually in `pebble.Client._request` or `_request_raw`?

Related: I thought we decided on just tracing in the one place for these (subprocess.run for hook tools and pebble request for Pebble stuff), or am I remembering wrong?